### PR TITLE
Ca version

### PR DESCRIPTION
**Pull request by cletusajibade**
Hello Sir, kindly merge my sources to the main repo

### CODE REVIEW

File: ca/src/test/java/com/cletus/app/QueryProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.cletus.app.QueryProcessor.stripStopWords;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class QueryProcessorTest {
```
Comment (Matthew-Mosior) [inline ca/src/test/java/com/cletus/app/QueryProcessorTest.java]: Try for consistent whitespace between end of imports and beginning of classes.

File: tests/cases_BKP.tsv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en	the walking dead	walking dead
+ga	Na daoine maithe	daoine maithe
+fr	plus ça change plus c'est la même chose	plus ça change plus c'est même chose
```
Comment (Matthew-Mosior) [inline tests/cases_BKP.tsv]: Please add additional updated test cases.

File: ca/src/test/java/com/cletus/app/QueryProcessorTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Test
```
Comment (Matthew-Mosior) [inline ca/src/test/java/com/cletus/app/QueryProcessorTest.java]: Good job keeping modularity in mind.

File: ca/src/test/java/com/cletus/app/QueryProcessorTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+package com.cletus.app;
+
+import org.junit.jupiter.api.Test;
```
Comment (Matthew-Mosior) [inline ca/src/test/java/com/cletus/app/QueryProcessorTest.java]: I'm getting some errors still with this import.  What is the recommended way to install it?

Thanks!

File: ca/src/main/java/com/cletus/app/QueryProcessor.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package com.cletus.app;
+
+import java.io.*;
+import java.util.*;
+
+public class QueryProcessor {
+
+    public static String stripStopWords(String lang, String query) {
```
Comment (Matthew-Mosior) [inline ca/src/main/java/com/cletus/app/QueryProcessor.java]: Might want to think about how you would handle situtations with lowercase and uppercase mixed together (for the stop words).

Here's an example:  The dog aNd the cat -> dog cat

File: tests/cases_BKP.tsv (hunk 2 above)
Comment (cletusajibade) [reply tests/cases_BKP.tsv]: Now uses the big cases.tsv file.

File: ca/src/test/java/com/cletus/app/QueryProcessorTest.java (hunk 4 above)
Comment (cletusajibade) [reply ca/src/test/java/com/cletus/app/QueryProcessorTest.java]: This has been updated, new code does not use any dependencies anymore

File: ca/src/main/java/com/cletus/app/QueryProcessor.java (hunk 5 above)
Comment (cletusajibade) [reply ca/src/main/java/com/cletus/app/QueryProcessor.java]: Handled in the new code. Code only converts to lower case internally to check correctness, does not affect the output.

**Pull request closed without merging**